checker/src/main/java/org/checkerframework/checker/signedness/SignednessAnnotatedTypeFactory.java

```diff
@@ -285,6 +288,16 @@ public Void visitCompoundAssignment(CompoundAssignmentTree tree, AnnotatedTypeMi
     }
   }
 
+  @Override
+  protected void adaptGetClassReturnTypeToReceiver(
+      AnnotatedExecutableType getClassType, AnnotatedTypeMirror receiverType, ExpressionTree tree) {
+    super.adaptGetClassReturnTypeToReceiver(getClassType, receiverType, tree);
+    final AnnotatedDeclaredType returnAdt = (AnnotatedDeclaredType) getClassType.getReturnType();
```

Comment:
A comment like the one in `NullnessAnnotatedTypeFactory.adaptGetClassReturnTypeToReceiver` would be useful here, I think.